lib/src/app_auth.dart

```diff
@@ -318,3 +313,11 @@ class InvalidSessionError implements Exception {
   @override
   String toString() => "Invalid session: $message";
 }
+
+/**
+ * API is invalid
+ */
```

Comment:
In general I feel like it's better to omit such overly generic comments. There's no additional information here that's not already conveyed in the class name.